### PR TITLE
SM-81 Moved store folders from screens to global store folder

### PR DESCRIPTION
**Pull request by artur-h**

### CODE REVIEW

File: src/store/auth/sagas/tests/rootAuthSaga.test.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { takeEvery } from 'redux-saga/effects';
 
-import { rootAuthSaga } from '~/screens/Auth/store/sagas';
-import { loginSaga } from '~/screens/Auth/store/sagas/loginSaga';
-import { loginUserAsync } from '~/screens/Auth/store';
+import { rootAuthSaga } from '~/store/auth/sagas';
```
Comment (Mykhailo-Kaduk) [inline src/store/auth/sagas/tests/rootAuthSaga.test.ts:3]: plaese change those import statements for norma ones. IN a project, it is allowed to make pathes one step back "../" or go from root via "~/"

File: src/store/user/sagas/tests/rootUserSaga.test.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { takeEvery } from 'redux-saga/effects';
 
-import { rootUserSaga } from '~/screens/User/store/sagas';
-import { fetchUserAsync } from '~/screens/User/store';
-import { fetchUserSaga } from '~/screens/User/store/sagas/fetchUserSaga';
+import { rootUserSaga } from '~/store/user/sagas';
+import { fetchUserAsync } from '~/store/user';
+import { fetchUserSaga } from '~/store/user/sagas/fetchUserSaga';
```
Comment (Mykhailo-Kaduk) [inline src/store/user/sagas/tests/rootUserSaga.test.ts:5]: same here

File: src/store/auth/sagas/tests/rootAuthSaga.test.ts (hunk 1 above)
Comment (artur-h) [reply src/store/auth/sagas/tests/rootAuthSaga.test.ts]: Fixed

File: src/store/user/sagas/tests/rootUserSaga.test.ts (hunk 2 above)
Comment (artur-h) [reply src/store/user/sagas/tests/rootUserSaga.test.ts]: Done

**Review by Mykhailo-Kaduk — APPROVED** [review]

**Review by andriibodryi — APPROVED** [review]

**Pull request merged**